### DISCUSSION TITLE
Does UpdateRelayFee now prevent per-channel fees to a single node?

### DISCUSSION THREAD

**Discussion by dscotese** [ACINQ/eclair#2003] [Q&A] [answered]
The release notes for 0.6.2 explain that updateRelayFee now accepts nodeId and nodeIds, and neither channelId nor channelIds now, which I think means that if you have two channels with the same peer, you can't specify different fees, for example to attract traffic to whichever needs it the most.  I suppose that's pretty rare, but I can imagine it happening.

**Comment by t-bast** [comment, marked as answer]
It simply doesn't make any sense to have different fees. Your channels to a given peer are interchangeable because of non-strict forwarding (https://github.com/lightningnetwork/lightning-rfc/blob/master/04-onion-routing.md#non-strict-forwarding).

Otherwise you're showing a completely incoherent state to the network and won't attract payments.
